config/console.php

```diff
@@ -2,25 +2,23 @@
 
 declare(strict_types=1);
 
-use Psr\Container\ContainerInterface;
+use Yiisoft\Di\ReferencesArray;
 use Yiisoft\Yii\Debug\Debugger;
-use Yiisoft\Yii\Debug\DebuggerIdGenerator;
-use Yiisoft\Yii\Debug\Storage\StorageInterface;
 
 if (!(bool)($params['yiisoft/yii-debug']['enabled'] ?? false)) {
     return [];
 }
 
 return [
-    Debugger::class => static function (ContainerInterface $container) use ($params) {
-        $params = $params['yiisoft/yii-debug'];
-        return new Debugger(
-            $container->get(DebuggerIdGenerator::class),
-            $container->get(StorageInterface::class),
-            array_map(
-                static fn ($class) => $container->get($class),
-                array_merge($params['collectors'], $params['collectors.console'] ?? [])
-            )
-        );
-    },
+    Debugger::class => [
+        '__construct()' => [
+            'collectors' => ReferencesArray::from(
+                array_merge(
+                    $params['yiisoft/yii-debug']['collectors'],
+                    $params['yiisoft/yii-debug']['collectors.web'] ?? []
```

Comment:
`collectors.web` – is it a typo?

Reply:
No. https://github.com/yiisoft/yii-debug/blob/9dccc68c34c43a12ec0eac13fbc492a361f78465/config/params.php#L37-L42

Reply:
Should it be `collectors.console`?

Reply:
Fixed.